fatrace.c

```diff
@@ -262,25 +323,30 @@ print_event (const struct fanotify_event_metadata *data,
             got_procname = true;
         } else {
             debug ("failed to read /proc/%i/comm", data->pid);
+            add_problem("Failed to read /proc/PID/comm, cannot determine comm.");
```

Comment:
I realize that my original code should have added an `%m` to get the actual error message. But this add_problem() call feels useless: you already see that the json line doesn't have `comm`, so it's already clear that its determination failed. But the problem message still doesn't tell you why.

Reply:
False. If the cached `comm` is usable it'll be included in the JSON object with no indication that it's a good guess rather than a value successfully read. In such case, this `add_problem` is the only indication that reading `comm` failed.

Reply:
Yes, that's why I added the `"However, cached comm is usable."` message.